Разметка всех страниц

rushe4ka opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/488405/tasks/4)

rushe4ka committed:
Добавляет основную разметку index.html

rushe4ka committed:
Добавляет основную разметку catalog.html

rushe4ka committed:
Update index.html

- убрала лишние комментарии
- заменила разметку поля авторизации пользователя на список
- разметила слайдер

rushe4ka committed:
Update catalog.html

- заменила разметку авторизации в шапке
- убрала лишние комментарии
- изменила разметку каталога

rushe4ka committed:
Размечает формы

rushe4ka committed:
Размечает адресные блоки

rushe4ka committed:
Размечает изображения и ссылки

- проставила размеры изображений в каталоге
- добавила ссылки на логотип Device
- добавила ссылки на соцсети и разработчика в футере

rushe4ka committed:
Исправляет разметку

- изменила разметку логотипа в хедере и футере
- изменила разметку списка категорий товаров в index.html
- изменила разметку списка транспортных компаний index.html
- изменила разметку ссылки в шапке "сравнить"
- изменила разметку формы "стоимость" catalog.html

rushe4ka committed:
Добавляет разметку дополнительных слайдов

-

valyavidyaeva commented:
Есть критичные ошибки, которые нужно доработать. 
Также очень важно соблюдать аккуратность кода и табуляцию, так будет легче читать код.

1. Поправить ошибки с раскладкой в ссылках в шапке, оформить пункты меню ссылками
![1](https://user-images.githubusercontent.com/19204483/170089453-cf5e7615-f030-47ab-8d6d-ec0174d156e4.jpg)


2. Заменить заголовки h1 на h2. Доработать верстку слайдеров.
![2](https://user-images.githubusercontent.com/19204483/170089682-09b46d14-c28a-4fc4-8a79-b6147eea364e.jpg)

3. Доработать секции "О нас", "Контакты" и "Подписка"
![3](https://user-images.githubusercontent.com/19204483/170089927-7c99bcf2-4da8-427e-98d5-1f56a07ba6bf.jpg)
![4](https://user-images.githubusercontent.com/19204483/170090008-c14dd54d-b775-485f-93ff-9424c0fd25c9.jpg)

4. Доработать ссылки в футере
![5](https://user-images.githubusercontent.com/19204483/170090037-cff3f6e4-2ce8-4c8f-bba2-be11be5f5e94.jpg)
 
5. Исправить ошибку в радио-кнопках. 
6. Доработать навигацию-крошки
![6](https://user-images.githubusercontent.com/19204483/170090171-0069542d-5962-4df1-a118-5624118a90b2.jpg)

7. Доработать блок с сортировкой товаров в каталоге
8. Оформить товары списком
9. В карточки добавить ссылки
![7](https://user-images.githubusercontent.com/19204483/170090756-e60ea672-a546-4344-97dc-83ab5b136863.jpg)

rushe4ka committed:
Изменяет разметку блока с соцсетями в футере

rushe4ka committed:
Исправляет ошибки разметки слайдер, tab, h1

- исправила ошибки табуляции
- изменила разметку хедера: кнопки входа и пункты списка
- изменила нумерацию заголовков
- изменила размертку слайдера index
- добавила разметку точек на слайдере

rushe4ka committed:
Добавляет разметку блока сервисов

rushe4ka committed:
Добавляет разметку блока сервисов

rushe4ka committed:
Изменяет разметку ссылок в футере

rushe4ka committed:
Исправляет разметку блока "О нас"

rushe4ka committed:
Исправляет ошибки в формах фильтров

rushe4ka committed:
Исправляет разметку крошек и сортировки

- изменила разметку сортировки товаров в каталоге
- добавила разметку хлебных крошек

rushe4ka committed:
Update catalog.html

- исправила разметку сортировок
- изменила разметку карточек товаров

rushe4ka committed:
Добавляет разметку пагинатора

rushe4ka committed:
Размечает атрибуты alt у изображений

rushe4ka committed:
Размечает классы и кнопок и списка категорий

rushe4ka committed:
Добавляет стрелки сортировки

rushe4ka committed:
Исправляет ошибки по валидатору

rushe4ka committed:
Update catalog.html

- разметка хлебных крошек
- разметка заголовков форм
- исправление ошибок в формах
- изменила разметку main
- разметила пагинатор

rushe4ka committed:
Переформатировала разметку каталога

rushe4ka committed:
Исправляет ошибки валидации

valyavidyaeva commented:
Очень хорошо поработала! Главный слайдер, табы про сервисы и фильтры сверстаны правильно.
Снова обращу внимание на табуляцию и пустые строки между блоками, важно соблюдать один стиль.
И ещё несколько мелких правок, чтобы сделать работу идеально!

1. Заголовок в шапке лишний, во-первых, он не отображает смысл контента, во-вторых, кнопка про каталог и так есть дальше.
![1](https://user-images.githubusercontent.com/19204483/170819246-8e1f9918-a3f8-42dd-acaa-56ae518bd740.jpg)

2. Вообще, шапка и футер на всех страницах должна быть одинаковая. Посмотри альты у логотипа и потерялась кнопка в форме поиска
![2](https://user-images.githubusercontent.com/19204483/170819349-6af75af5-e813-4023-929b-b65fd7eb34d4.jpg)

3. Если есть сортировка, то список должен быть упорядоченный – <ol>
![3](https://user-images.githubusercontent.com/19204483/170819352-a182bd5d-8fc4-4a6c-aeaa-b340851d8f27.jpg)

4. Доработать ссылки в футере на странице каталога.
![4](https://user-images.githubusercontent.com/19204483/170819363-c3766bb7-2c79-4b4b-a4ca-88bde0b6afbc.jpg)

rushe4ka committed:
Исправляет порядок атрибутов и форматирование тегов

rushe4ka committed:
Убирает лишний заголовок в шапке

rushe4ka commented:
Подпись в конце описания должна иметь точку. Точка нужна для интонационной паузы при чтении скринридером.

Это из кодгайда. Критерий ALLY-01. Я поэтому везде поставила точки

rushe4ka committed:
Исправляет ошибки по комментариям

rushe4ka committed:
Update .editorconfig

rushe4ka committed:
Изменяет разметку крошек и определений

rushe4ka committed:
Экспортирует графику в проект

rushe4ka committed:
Подключает графику в разметку

rushe4ka committed:
Исправляет ошибку с номером

rushe4ka committed:
Update .editorconfig

rushe4ka committed:
Исправляет атрибут ссылки в футере

rushe4ka committed:
исправляет разметку обертки в шапке
